isMobile.js

```diff
@@ -95,13 +94,20 @@
         };
         this.seven_inch = match(seven_inch, ua);
         this.any = this.apple.device || this.android.device || this.windows.device || this.other.device || this.seven_inch;
-
         // excludes 'other' devices and ipods, targeting touchscreen phones
         this.phone = this.apple.phone || this.android.phone || this.windows.phone;
-
         // excludes 7 inch devices, classifying as phone or tablet is left to the user
         this.tablet = this.apple.tablet || this.android.tablet || this.windows.tablet;
 
+        this.desktop = !this.tablet && !this.phone;
```

Comment:
I'd prefer to keep the library limited to doing only one thing: try to identify a mobile device. I don't think we should assert that everything else not covered is a desktop; it could be a new mobile device, a console, TV, etc.